src/components/Intercom/SetupIntercom.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "react";
+import { useUser } from "../../context";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export default function SetupIntercom({ children }: Props) {
+  const { user } = useUser();
+
+  useEffect(() => {
+    if (typeof window === "undefined") {
+      return;
+    }
+
+    window.Intercom("boot", {
+      app_id: process.env.NEXT_PUBLIC_INTERCOM_APP_ID,
+      email: user?.email,
```

Comment:
Need the name, clerk org name, id and clerk user id

Reply:
Organization ID and Name might need to be an array, since a user can have multiple and we don't want to override them when they change organization